src/Field/Field.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react'
+import styled from 'styled-components'
+import { darken } from 'polished'
+
+import { Icon } from '../Icon'
+import { Text } from '../Text'
+import { Box } from '../Box'
+import { theme } from '../theme'
+
+type Props = {
+  children: React.ReactElement
+  showCaret?: boolean
+  renderAsTitle?: boolean
+  className?: string
+  id: string
+  error?: boolean // nb this needs to be optional for legacy support - but we will want to enforce in  the future
+  label?: string
+  outlined?: boolean
+  value: string
+  trailingIcon?: string
+  errorMsg?: string
+  dropdownKey?: string // nb only used by dropdown component but required on content in that context
+  required?: boolean // also optional for now to maintain backwards compaibility
+  fullHeight?: boolean // used to make text areas full size
+}
+
+export const Field = ({
+  children,
+  renderAsTitle,
+  className = '',
+  error,
+  id,
+  label,
+  outlined = false,
+  value,
+  trailingIcon,
+  errorMsg,
+  dropdownKey,
+  required,
+  showCaret,
+  fullHeight = false,
+}: Props) => {
+  return (
+    <Container className={className}>
+      {label && (
+        <Box mb={outlined ? '4px' : '0px'}>
```

Comment:
I think the spacing issue between the asterisk and the title on the `Confirmation` component needs to be addressed here, not sure where exactly without testing

Reply:
@josianed happy for you to pick this up as referenced above, if you've have more context 👍🏻